### DISCUSSION TITLE
Admin management with Web page builder

### DISCUSSION THREAD

**Discussion by mobiliseapplabllp** [givanz/Vvveb#192] [Q&A]
Hi, 

We plan to integrate a Web page builder into our system. I want to find out if it comes with an Admin panel also. or I am missing something

I have seen one of the demos where the admin panel is shown which list the pages created:

https://docs.vvveb.com/media/docs/menus.png

I am using editor, can anyone pls guide?

Thank you!

**Comment by givanz** [comment]
Hi

VvvebJs page builder does not have an admin panel.

The screenshot is from [Vvveb CMS](https://github.com/givanz/Vvveb) admin that has VvvebJs page builder integrated.

If you want to use the admin template without the CMS you can download it from https://github.com/givanz/vvveb-admin-template

**Comment by mobiliseapplabllp** [comment]
Dear Givanz,

You are doing such awesome work, and I am so glad to hear from you.
I understand that the screenshot is from CMS, but I am referring to the following screenshot, which, when we click on the Design button, takes us to EDITOR.

<img width="1351" alt="Screenshot 2024-09-23 at 16 31 39" src="https://github.com/user-attachments/assets/23a74c07-70f6-4f3a-8708-671ad982ed83">

I wanted to know how to integrate this section in my application.

I am trying hard to integrate your editor into my application, and I have a couple of more questions. With your permission, can i include them here, or Should i open up a new thread? Kindly guide.

I owe you a donation once i am successful in integrating this editor into my platform. I really appreciate the work you are doing.

Warm regards,
Ashish Sharma

**Comment by givanz** [comment]
Thank you.

The design url is pointing to the editor page passing the url of the page that is edited http://demo.vvveb.com/admin/?module=editor/editor&url=//demo.vvveb.com/hello-world-1-1&template=content/post.html

The editor.php sets current page to the page from url parameter https://github.com/givanz/Vvveb/blob/master/admin/controller/editor/editor.php#L189 

The pages list that shows in the file manager is then passed to VvvebJs page builder https://github.com/givanz/vvveb-admin-template/blob/faf193ebf4a9fad5ee5ba2e5f46af55d6f45f1d5/editor/editor.html#L3373

**Comment by mobiliseapplabllp** [comment]
Dear Givaz,

I hope you're doing well.
We are currently using the PHP version of the Vvveb editor, which we installed from GitHub. However, we have encountered an issue where certain blocks available in the demo version, accessible at the link below, are missing from our setup:

https://demo.vvveb.com/admin/?module=/editor/editor&template=index.html

[Image]

As these blocks are crucial for our project, we would greatly appreciate your guidance on how to import or enable them in our version. We have already integrated this builder deeply into our system, so resolving this issue is essential for our progress.

Your support would be invaluable, and we are eager to hear any suggestions or solutions you may have.

Thank you in advance for your time and assistance. We are hoping for your prompt help on this matter.

Warm regards,
Ashish Sharma


From: Givan ***@***.***>
Date: Saturday, 28 September 2024 at 14:55
To: givanz/Vvveb ***@***.***>
Cc: ASHISH SHARMA ***@***.***>, Author ***@***.***>
Subject: Re: [givanz/Vvveb] Admin management with Web page builder (Discussion #192)

Thank you.

The design url is pointing to the editor page passing the url of the page that is edited http://demo.vvveb.com/admin/?module=editor/editor&url=//demo.vvveb.com/hello-world-1-1&template=content/post.html

The editor.php sets current page to the page from url parameter https://github.com/givanz/Vvveb/blob/master/admin/controller/editor/editor.php#L189

The pages list that shows in the file manager is then passed to VvvebJs page builder https://github.com/givanz/vvveb-admin-template/blob/faf193ebf4a9fad5ee5ba2e5f46af55d6f45f1d5/editor/editor.html#L3373

**Reply by givanz** [reply]
https://github.com/givanz/VvvebJs/issues/385 (duplicate)
